interface/grpc/core/info.go

```diff
@@ -0,0 +1,29 @@
+package core
+
+import (
+	"context"
+
+	"github.com/mesg-foundation/core/config"
+	"github.com/mesg-foundation/core/protobuf/coreapi"
+)
+
+// Info returns all necessary informations from the core.
+func (s *Server) Info(ctx context.Context, request *coreapi.InfoRequest) (*coreapi.InfoReply, error) {
+	c, err := config.Global()
```

Comment:
Do not use config.Global - pass the config or needed data

Reply:
What would be your recommendations ?
- Give the config as dependency of `Server` (we need to change the `New` because we give the address and this address comes from the config)
- Delegate to the API and give the config to the API.
- Give a struct with all the info (needed by the Info api) to the server when we create it
- Keep it like that, `config.Global` is a singleton so let's use a singleton

I'm not really happy with any of them so I'm open for this but anyway this might be done in another PR.